apps/test/unit/templates/rubrics/AiAssessmentFeedbackTest.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {expect} from '../../../util/reconfiguredChai';
+import {shallow} from 'enzyme';
+import AiAssessmentFeedback from '@cdo/apps/templates/rubrics/AiAssessmentFeedback';
+
+describe('AiAssessmentFeedback', () => {
+  const props = {
+    learningGoalKey: 'learning_key',
+  };
+
+  it('displays checkboxes when thumbs down is clicked', () => {
+    const wrapper = shallow(<AiAssessmentFeedback {...props} />);
+    wrapper.find('input').at(1).simulate('change');
+    expect(wrapper.find('Checkbox')).to.have.lengthOf(4);
+  });
+
+  it('displays textbox when other checkbox is selected', () => {
```

Comment:
nit: I wonder if it is better to say `when thumbs up checkbox is selected` instead of `other checkbox` just to make it a bit clearer?

Reply:
This test is for making sure the textbox appears when the checkbox labelled "Other" is checked, I can reword this

Reply:
Addressed with [cad0179](https://github.com/code-dot-org/code-dot-org/pull/54634/commits/cad01798c831b16c560d446ffe43da23fe6dd853)